kannon/master.py

```diff
@@ -5,6 +5,7 @@
 import logging
 
 import gokart
+from luigi.task import flatten
```

Comment:
same handling with luigi:  https://github.com/spotify/luigi/blob/v1.2.1/luigi/task.py

It seems to public, documented function.

https://luigi.readthedocs.io/en/stable/api/luigi.task.html#luigi.task.flatten